docs/release-info/migr-ansi-sql-null.md

```diff
@@ -0,0 +1,266 @@
+---
+id: migr-ansi-sql-null
+title: "Migration guide: SQL compliant mode"
+sidebar_label: SQL compliant mode
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+-->
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+In Apache Druid 28.0.0, the default [null handling](../querying/sql-data-types.md#null-values) mode changed to be compliant with the SQL standard:
+
+The SQL standard defines any comparison to null to be unknown.
+Therefore, according to this three-value logic, `x <> 'some value'` only returns non-null values.
+
+Now, Druid stores segments in a SQL compatible null handling mode by default.
+
+The default Druid configurations for SQL compatible null handling mode is as follows:
+
+* `druid.generic.useDefaultValueForNull=false`
+* `druid.expressions.useStrictBooleans=true`
+* `druid.generic.useThreeValueLogicForNativeFilters=true` 
+
+Note Druid has always applied three-value logic by default to expressions.
+Therefore, queries such as `(x+y) <> ‘some value’` exclude null values even prior to Druid 28.0.0.
+
+At query time, Druid treats data from segments written with the legacy two-value logic as follows:
```

Comment:
given that we discuss the legacy mode right after this, i wonder if it is needed since these seem like obvious statements.